src/components/CoreLabel/Styles.ts

```diff
@@ -3,11 +3,23 @@ import styled from 'styled-components';
 import {fonts} from 'styles';
 
 export const Container = styled.div`
-  display: flex;
+  width: 100%;
 `;
 
+export const Box = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin: 10px 0px 15px 0px;
+  width: 100%;
+`;
```

Comment:
Let's keep following the convention of sorting lines in asc order.
I use this VSCode setting to sort lines: 
<img width="601" alt="image" src="https://github.com/thenewboston-developers/thenewboston-Frontend/assets/52558124/80629ccb-b9e2-4721-8a7a-3963336bdb7d">

But I think @buckyroberts would have a better option/suggestion since my lines also aren't always well sorted :P